src/logging.cpp

```diff
@@ -203,9 +203,9 @@ std::string BCLog::Logger::LogTimestampStr(const std::string& str)
             strStamped.pop_back();
             strStamped += strprintf(".%06dZ", nTimeMicros%1000000);
         }
-        int64_t mocktime = GetMockTime();
-        if (mocktime) {
-            strStamped += " (mocktime: " + FormatISO8601DateTime(mocktime) + ")";
+        std::chrono::seconds mocktime = GetMockTime();
+        if (mocktime.count() == 0) {
```

Comment:
Shouldn't that be `!=` instead of `==`? Or:

```suggestion
        if (mocktime > 0) {
```

Reply:
🤦‍♀️ good catch.

Reply:
fixed. can't compare chrono time directly to an int, so I updated to `if (mocktime > 0s)` to keep the comparison in chronos.